domains/noobchecker.json

```
@@ -0,0 +1,10 @@
+{
+  "repo": "1https://github.com/no0b-op/Cc.github.io",
+  "owner": {
+    "username": "no0b-op",
+    "email": "labani9668430706@gmail.com"
+  },
+  "record": {
+    "CNAME": "no0b-op.github.io/Cc.github.io"
```

CNAME can not have `/`
```suggestion
    "CNAME": "no0b-op.github.io"
```